src/wp-includes/class-wp-image-editor-imagick.php

```diff
@@ -220,6 +220,9 @@ public function set_quality( $quality = null ) {
 					}
 					break;
 				case 'image/avif':
+					$this->image->setOption( 'heic:speed', 7 );
```

Comment:
Would be great to add a comment with some context: What is `heic:speed` and why is it set to `7` here?

Reply:
added